txnsync/txngroups.go

```diff
@@ -103,7 +107,7 @@ func compressTransactionGroupsBytes(data []byte) ([]byte, error) {
 	b := make([]byte, 0, len(data))
 	_, out, err := compress.Compress(data, b, 1)
 	if err == nil && len(data) > len(out)*maxCompressionRatio {
-		return nil, errors.New("compression exceeded compression ratio")
+		return nil, fmt.Errorf("compression exceeded compression ratio: compressed data len: %d", len(out))
```

Comment:
this would create a warning, which isn't what we want. An info message is sufficient.